Core/clim-core/presentation-defs.lisp

```diff
@@ -1285,7 +1287,10 @@ protocol retrieving gestures from a provided string."))
 
 (define-presentation-method accept ((type symbol) stream (view textual-view)
                                     &key)
-  (accept-using-read stream type))
+  (let* ((read-result (accept-using-read stream type)))
+    (if (and (null read-request) default)
```

Comment:
Do I have to submit a new commit, or do you have the ability to pull this with those changes?